kmip/core/messages/payloads/get.py

```diff
@@ -130,7 +128,7 @@ def read(self, istream):
         self.object_type.read(tstream)
         self.unique_identifier.read(tstream)
 
-        secret_type = self.object_type.enum
```

Comment:
The short reason is to make Enumeration consistent with the other primitives.

The longer (but still short) reason is that Enumeration used to be a subclass of Integer. Since Integer already defined "value", Enumeration had to use a different attribute to hold the enumeration value. Now that Enumeration is its own stand-alone class, it can use "value" without colliding with anything.

This simple switch caused a buuuuuunch of cascading changes, but I think its better for the long term.